frontend/src/scenes/surveys/SurveyOverview.tsx

```diff
@@ -94,6 +97,12 @@ export function SurveyOverview(): JSX.Element {
                         <SurveySchedule />
                     </div>
                 </div>
+                {featureFlags[FEATURE_FLAGS.SURVEYS_PARTIAL_RESPONSES] && (
+                    <div className="flex flex-col mt-4">
+                        <span className="card-secondary">Store partial responses</span>
+                        <span>{survey.store_partial_responses ? 'Yes' : 'No'}</span>
+                    </div>
+                )}
```

Comment:
<img width="295" alt="Screenshot 2025-03-08 at 09 00 20" src="https://github.com/user-attachments/assets/9f3b848a-2d69-4e35-9445-a67f02974f02" />

our toggles always look like this, so I'd not say yes/no but rather give a title and the toggle itself

> Store partial responses

maybe just 'Enable partial responses' is better?
Store is a bit confusing, enable as a feature makes more sense to me, the title can link to the docs explaining how this works

Reply:
here is just the summary, so this comment is more about where we define the radio button, but the overview label should reflect the text we put in there

Reply:
i agree, didn't know about the Switch. changed to that, added screenshots in the PR